src/styles/_components.scss

```diff
@@ -150,13 +150,20 @@
 
 
 .weave-list-item {
```

Comment:
While you're at it, maybe you could also make this chunk a styled component? :)

Reply:
:see_no_evil: might tackle this in a separate PR!